nominal/nominal.py

```diff
@@ -292,6 +292,23 @@ def download_attachment(rid: str, file: Path | str) -> None:
     attachment.write(Path(file))
 
 
+def upload_video(
+    file: Path | str, name: str, start: datetime | str | IntegralNanosecondsUTC, description: str | None = None
+) -> Video:
+    """Upload a video to Nominal from a file."""
+    conn = get_default_client()
+    path = Path(file)
+    file_type = FileType.from_path(path)
+    with open(file, "rb") as f:
+        return conn.create_video_from_io(f, name, _parse_timestamp(start), description, file_type)
```

Comment:
I think `mimetypes.guess_type(path)` should work for other types of video containers aside from mp4 but maybe worth a test? We don't really enforce anything for container type on the backend (though maybe we should lol)

Reply:
yes, it does!